.ci_cd/aws/cloudformation-aws-marketplace.yaml

```diff
@@ -0,0 +1,501 @@
+          - SSHLocation
+          - CIDRBlock
+      - Label:
+          default: OpenRemote Configuration
+        Parameters:
+          - Password
+      - Label:
+          default: E-mail Configuration
+        Parameters:
+          - SMTPHost
+          - SMTPUser
+          - SMTPPassword
+          - SMTPPort
+          - SMTPTLS
+          - SMTPFrom
+          - SMTPProtocol
+
+Parameters:
+    Name:
+      Description: The name for this EC2 Instance on the EC2 Overview page
+      Type: String
+      Default: OpenRemote
+    Instance:
+      Description: Choose a instance based on your monthly budget
+      Type: String
+      Default: Medium
+      AllowedValues:
+        - Small
+        - Medium
+        - Large
+    Hostname:
+      Description: Optionally fill in the FQDN (Fully Qualified Domain Name) that you want to use for this OpenRemote instance
+      Type: String
+    Keypair:
+      Description: Choose a keypair for SSH Access
+      Type: AWS::EC2::KeyPair::KeyName
+    CIDRBlock:
+      Description: Please set CIDR to x.x.x.x/32 to allow one specific IP address access, 0.0.0.0/0 to allow all IP addresses access, or another CIDR range
+      Type: String
+      AllowedPattern: (\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})/(\d{1,2})
+    SSHLocation:
+      Description: Fill in your Home IP for SSH Access. Set CIDR to x.x.x.x/32.
+      Type: String
+      AllowedPattern: (\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})/(\d{1,2})
+      ConstraintDescription: Must be a valid IP CIDR range of the form x.x.x.x/x.
+    Password:
+      Description: Create an (admin) password for your OpenRemote instance
+      Type: String
+      MinLength: 8
+      AllowedPattern: ^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[^0-9a-zA-Z])(?=\S+$).{8,}$
+      ConstraintDescription: Password must contain 1 or more uppercase characters. Password must contain 1 or more digit characters. Password must contain 1 or more special characters. Password must be 8 or more characters in length.
+      NoEcho: true
+    SMTPHost:
+      Description: Fill in the SMTP Hostname for sending e-mails
+      Type: String
+    SMTPUser:
+      Description: Fill in the SMTP username for sending e-mails
+      Type: String
+    SMTPPassword:
+      Description: Fill in the SMTP password for sending e-mails
+      Type: String
+      NoEcho: true
+    SMTPPort:
+      Description: Fill in the SMTP port for sending e-mails
+      Type: Number
+      Default: 587
+    SMTPTLS:
+      Description: Select if you want to use TLS for sending e-mails
+      Type: String 
+      AllowedValues:
+        - true
+        - false
+      Default: true
+    SMTPFrom:
+      Description: Fill in the From e-mail address you want to use for sending e-mails
+      Type: String
+    SMTPProtocol:
+      Description: Select the SMTP protocol you want to use for sending e-mails (smtps = SSL)
+      Type: String
+      AllowedValues:
+        - smtp
+        - smtps
+      Default: smtps
```

Comment:
Would be nice if it is clear that most of these parameters are optional. Users may otherwise quickly get overwhelmed and abort the wizard and not try running OpenRemote at all.